firebase-installations/api.txt

```diff
@@ -94,6 +94,7 @@ package com.google.firebase.installations.remote {
     ctor public InstallationResponse();
     method @NonNull public static com.google.firebase.installations.remote.InstallationResponse.Builder builder();
     method @NonNull public abstract InstallationTokenResult getAuthToken();
+    method @NonNull public abstract String getFid();
     method @NonNull public abstract String getName();
```

Comment:
I recommend calling this "uri" instead of "name" since that is a more precise description than "name".
On the server-side we have to call this field "name" due to API design restrictions, but the value of the field is the unique resource identifier (URI) of this Firebase Installation.

Reply:
Done. Thanks.